sdk/src/dlob/DLOBSubscriber.ts

```diff
@@ -0,0 +1,64 @@
+import { UserMap } from '../userMap/userMap';
+import { DLOB } from './DLOB';
+import { SlotSubscriber } from '../slot/SlotSubscriber';
+import { EventEmitter } from 'events';
+import StrictEventEmitter from 'strict-event-emitter-types';
+
+type DLOBSubscriptionConfig = {
+	userMap: UserMap;
```

Comment:
these could be swapped our for a DLOBSource and SlotSource interface so you can modify where the DLOB is built from and how the Slot is calculted